2-mandatory/3-stocks.js

```diff
@@ -48,6 +58,18 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
+     const priceChanges = [];
+
+  for (let i = 0; i < closingPricesForAllStocks.length; i++) {
```

Comment:
Nice work!
I like the way you're using extra variables to make the code very clear and easy to read.